New package: Drifters v0.6.0

JuliaRegistrator opened the pull request:
- Registering package: Drifters
- Repository: https://github.com/JuliaClimate/IndividualDisplacements.jl
- Created by: @gaelforget
- Version: v0.6.0
- Commit: 53db417bb3415c0962dc600a52e2697e0e28afb7
- Reviewed by: @gaelforget
- Reference: https://github.com/JuliaClimate/IndividualDisplacements.jl/issues/3#issuecomment-2479960204
- Description: Trajectory simulations for point particles in the Ocean & Atmosphere
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598753748b4892485ca904ec8050f8e2ca77c1a8db8199bd444ac54c91ceef2ade8e63cb8118491c30ed4fb5e17b2677a67539be48b6686ceaeca33f85c833de123beaf7e1a8b82e5a4ddd4517e537d188c063d5149ceb0ee6142d89cb36d6e8dfb94df6196331b824778fb11d0f2b02b09e4038bdbcddc76e88fdabe3ac8d6f5d435691633b3370edb7e222252460bca62 -->

gaelforget commented:
this is a rename

gaelforget commented:
[noblock]

gaelforget commented:
I do not understand this : 

`Repo URL does not end with /name.jl.git, where name is the package name`

Given that [Drifters/Package.toml](https://github.com/JuliaRegistries/General/pull/119126#diff-dbc4dc76d03a422cde7fbd77b5b7e455e82a357bcb0b994c85f5f509aef23994) has 

`repo = "https://github.com/JuliaClimate/IndividualDisplacements.jl.git"`

And https://github.com/JuliaRegistries/General/pull/119150 has 

`repo = "https://github.com/JuliaClimate/Drifters.jl.git"`

What am I expected to do here?

gaelforget commented:
I looked for a recently merged example and found 

https://github.com/JuliaRegistries/General/pull/119402

then 

https://github.com/JuliaRegistries/General/pull/119407

where 

`repo = "https://github.com/Elijah-Phifer/PlanetSim.jl.git"`

so I am going to try renaming here with

`repo = "https://github.com/JuliaClimate/Drifters.jl.git"`

gaelforget commented:
renamed repo back to IndividualDisplacements.jl [noblock]

JuliaRegistrator committed:
New package: Drifters v0.6.0

UUID: bd752fb7-3f37-44cb-a8fb-f461137b623f
Repo: https://github.com/JuliaClimate/IndividualDisplacements.jl.git
Tree: 0a7dfc1c008e101f90f2a4078883c93120ddcb90

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

goerz commented:
Closing in favor of https://github.com/JuliaRegistries/General/pull/119538

GunnarFarneback commented:
[noblock]
Given that the PR description says
> `Registering package: Drifters`
> `Repository: https://github.com/JuliaClimate/IndividualDisplacements.jl`

it appears that when the registration was triggered, the repository was still called `IndividualDisplacements.jl`, or at least that is the name of the repository picked up by Registrator. AutoMerge reacts to the disparity between then package name and the repository name.

My best suggestion is to again rename the repository to `Drifters.jl` and afterwards retrigger the registration. (It will open a different PR in this case, but that is unavoidable since the repository name is part of the branch naming done by Registrator.)

gaelforget commented:
> [noblock] Given that the PR description says
> 
> > `Registering package: Drifters`
> > `Repository: https://github.com/JuliaClimate/IndividualDisplacements.jl`
> 
> it appears that when the registration was triggered, the repository was still called `IndividualDisplacements.jl`, or at least that is the name of the repository picked up by Registrator. AutoMerge reacts to the disparity between then package name and the repository name.
> 
> My best suggestion is to again rename the repository to `Drifters.jl` and afterwards retrigger the registration. (It will open a different PR in this case, but that is unavoidable since the repository name is part of the branch naming done by Registrator.)

I remember that the `if` in 

`if you rename the repository so it has a new URL, make a PR to edit the URL stored in the registry for the old package name to point to the new URL ([example](https://github.com/JuliaRegistries/General/pull/40190/files)). This allows the old versions of the package under the previous name to continue to work.`

confused me when I did this. I think at first I did not rename the repo, then I thought maybe I actually needed to, and opened https://github.com/JuliaRegistries/General/pull/119150
